docs/sources/enterprise/aws-secrets-plugin.md

```diff
@@ -0,0 +1,87 @@
+---
+aliases:
+  - /docs/grafana/latest/enterprise/aws-secrets-plugin/
+description: 'Configuring the AWS Secrets Manager plugin'
+keywords:
+  - grafana
+  - secrets
+  - secretsmanager
+  - aws
+  - backend
+title: AWS Secrets Manager Integration
+weight: 2000
+---
+
+# AWS Secrets Manager Plugin
+
+By default, Grafana stores all encrypted secrets in its own SQL database. The AWS Secrets Manager plugin is available for users who wish to store secrets in [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/). In the current version of Grafana, this is limited to datasource secrets, but it will eventually include all secrets (e.g. user credentials).
+
+There are a number of configuration options available that determine secret storage behavior in AWS Secrets Manager. We also provide the ability to migrate secrets from existing Grafana instances to AWS Secrets Manager. We provide a degree of backwards compatibility so that the plugin can be quickly uninstalled, but it has some [limitations](#limitations).
+
```

Comment:
Other similar documentation lists prerequisites for using their related plugins or services. See https://grafana.com/docs/grafana/next/setup-grafana/configure-security/configure-database-encryption/encrypt-secrets-using-aws-kms/ for an example.

Also, add a note specifying that this is available in Grafana Enterprise:

```suggestion

> **Note:** Available only in [Grafana Enterprise]({{< relref "/enterprise/" >}}).